test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -7,9 +7,10 @@
 
 #include "test_ucp_memheap.h"
 
-#include <ucs/sys/sys.h>
 extern "C" {
+#include <ucp/core/ucp_context.h>
 #include <ucp/core/ucp_mm.h> /* for UCP_MEM_IS_ACCESSIBLE_FROM_CPU */
+#include <ucs/sys/sys.h>
```

Comment:
why need to add includes?

Reply:
to avoid this
```
ucp/test_ucp_rma.cc:120:30: error: invalid use of incomplete type ‘ucp_config_t {aka struct ucp_config}’
             if (!m_ucp_config->ctx.proto_enable &&
                              ^
In file included from /labhome/mikhailb/wgit/ucx-tmp/src/ucp/api/ucp.h:12:0,
                 from ucp/ucp_test.h:10,
                 from ucp/test_ucp_memheap.h:11,
                 from ucp/test_ucp_rma.cc:8:
/labhome/mikhailb/wgit/ucx-tmp/src/ucp/api/ucp_def.h:77:16: error: forward declaration of ‘ucp_config_t {aka struct ucp_config}’
 typedef struct ucp_config                ucp_config_t;

```